index.d.ts

```diff
@@ -0,0 +1,6 @@
+export default class AutoCheckElement extends HTMLElement {
+  readonly input?: HTMLInputElement;
```

Comment:
https://github.com/github/remote-input-element/pull/6#discussion_r306818969

```suggestion
  readonly input: HTMLInputElement | null;
```